### ISSUE TITLE
Update Legislative Affairs and Administration SharePoint Sites

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#17955]
> What application are you using?

Office 365 (Teams, Sharepoint, etc.)

> Describe the problem.

In collaboration with the Legislative Affairs Manager, I would like to update our Sharepoint home page to make it easier to navigate.

> Describe the outcome you'd like to see when this feature is implemented.

I like the "look" of TPW HR's page, with larger graphics, versus standard file folders. 

> Describe any workarounds you currently have in place or alternative solutions you've considered.

I'm not sure how much this space is currently being utilized, mostly internally within our own divisions. We have a lot of reference data that needs to be easily accessible to the whole department - RCA deadlines and templates, etc

> Is there anything else we should know?

Ideal to have the division homepage organized by the end of July, and a few division staff trained on how to manage the data within the space.  The most commonly used documents will be related to the RCA process, and after the August budget meetings, City Council meetings will be back on a regular schedule.

> Requested By
Susan D.

Request ID: DTS24-115740

**Comment by dianamartin** [comment]
# 6/24/24
Should we reach out to Susan to ask her what her goals are?

**Comment by dianamartin** [comment]
# 6/24/24
We let Stephanie from OPM know about this request and how it might impact her work on updating the TPW Admin Site for Grants with Andrei. 
- [x] She scheduled a meeting for all of us on Wed, July 10th

**Comment by KaroEngstrom** [comment]
### Fri 7/12/24 
- per meeting with Admin, Legislative Affairs, and OPM, we will: 
     - repurpose the existing tpwAdmin site for Legislative Affairs because most of the content there is theirs 
     - update the handle to a new slug tpwLegislativeAffairs (SN INC0410630 placed by @KaroEngstrom ) 
     - let OPM and Legislative Affairs know once that update has been done
     - update the name on the site ✅ 
     - update links in the tpwHub to reflect final URLs
     - create a separate site for tpwAdmin (hoping to repurpose that slug once it's been removed from the above-mentioned site) 
     - follow up with Susan on the plan for the new tpwAdmin site once that site has been created; previously created content [here](https://cityofaustin.sharepoint.com/:w:/r/sites/tpwDTS/Shared%20Documents/SharePoint%20Admin%20Docs/tpwAdmin_%20CAF%20text%20.docx?d=w401c3653768b40a4b27034e332eb00b2&csf=1&web=1&e=cYnK4V) 

Notes 
- @dianamartin  advised meeting attendees on SharePoint permissions best practices and cleaned up member roster during the meeting 
- OPM expressed interest in learning about building SharePoint pages during this process; will likely ask for DTS help whn needed

**Comment by KaroEngstrom** [comment]
### MOn 7/15/24 
- the handle has been updated to tpwLegislativeAffairs ✅ 
- notified the participants via the meeting chat ✅ 
- updated the above-linked text document to include the old tpwAdmin home page in case some of the content is needed for the new tpwAdmin site 
- CTM says that "after some time when everything is synced the old name should become available", regarding the tpwAdmin page name for a team site (might be available for a communication site earlier)

**Comment by KaroEngstrom** [comment]
## Wed 7/17/24 
- Legislative Affairs has been notified and the links are up-to-date on TPW Hub (the Administration* link to be updated once we have a new tpwAdmin site) 
- previous Administration page content is [here](https://cityofaustin.sharepoint.com/:w:/r/sites/tpwDTS/_layouts/15/Doc.aspx?sourcedoc=%7B401C3653-768B-40A4-B270-34E332EB00B2%7D&file=tpwAdmin_%20text.docx&action=default&mobileredirect=true)
- the PIR materials issue needs to be addressed with this | #17812

**Comment by dianamartin** [comment]
# 7/30/24
- I made a new SharePoint site: [tpw-Admin website](https://cityofaustin.sharepoint.com/sites/TPW-Admin)

**Comment by KaroEngstrom** [comment]
## Thu 8/15/24
- met with Admin PIR team and Susan; largely focused on the PIR content piece 
- this is also related to #17812 

### Notes
PIR 
* GovQA assigns to TPW PIR Team 
* Stephanie, Drew, Susan, Joana, Alyssa all get copied on new items that come in
* Alyssa assigns it out as the primary, Joana might step in if needed 
* Spreadsheet to keep track of what each division does 
* Every PIR has the same timeline, depending on the volume 
* Reasonable amount of time, but also depending on volume of what they’re asking for 
* Asking for scope, cost estimate, clarification 
* PIR in pause mode when asking for cost estimate (the clock stops) 
* Sends out to stakeholder - each division has told them who the point person is 
* Will send directly to a person if they’re named (copy SPOC) 
* Alyssa will create a folder on the G: drive
* Permissions in G: drive? 
    * This is an ongoing issue and seems unclear 
    * Open folders aren’t ideal for access but otherwise there’s no way to have people send large files 
* Stephanie - PWD legacy side 
    * Directly email to Stephanie under that model 
    * Jeanette had shown her how to do a folder 
* Needs everything in PDF forms - maybe we need training for the PIR SPOCs 
    * Needing to lock Excel spreadsheets, etc.

**Comment by KaroEngstrom** [comment]
@dianamartin - should we close this one and Susan/we will create a new issue when they have reviewed and are ready to proceed?

**Comment by dianamartin** [comment]
@KaroEngstrom I think we need to email Susan and say that we have an open request to get their SharePoint set-up. We understand they have to talk internally, but we're waiting on them to get back to us. Since she was the requester for this, I think we should keep this one open. Didn't Andrei have a different SharePoint issue open?

**Comment by KaroEngstrom** [comment]
Andrei only requested access in #18186 and all the Legislative Affairs notes are here. I can split into separate issues if needed. I'll email Susan to follow up.

**Comment by stephaniemoorer** [comment]
@dianamartin Andrei does have this issue #18447 which may be a SharePoint solution. 
Discovery meeting is scheduled for 8/27.

**Comment by dianamartin** [comment]
@stephaniemoorer Yeah, it's a new issue. We'll have to scope what it is they're trying to do with RCAs and figure out which platform makes sense for them.

**Comment by KaroEngstrom** [comment]
@dianamartin Should we close this? We've met with Admin and discussed options for SharePoint site and for PIR solution. Their team will review internally and is supposed to get back to us at unknown time. We could create a new issue at that time.

**Comment by dianamartin** [comment]
@KaroEngstrom Yes, I think so since they're no longer asking for our assistance on this.
